usaspending_api/awards/v2/filters/location_filter_geocode.py

```diff
@@ -0,0 +1,55 @@
+from django.apps import apps
+
+from usaspending_api.awards.models import Award, TransactionNormalized
+from usaspending_api.common.exceptions import InvalidParameterException
+
+loc_dict = {
+        'country': 'location_country_code',
+        'state': 'state_code',
+        'county': 'county_code',
+        'district': 'congressional_code'
+    }
+
+
+def geocode_filter_locations(scope, values, model):
+    """
+    Function filter querysets on location table
+    scope- place of performance or recipient location mappings
+    values- array of location requests
+    model- awards or transactions will create queryset for model
+    returns queryset
+    """
+    or_queryset = None
+
+    for v in values:
+        fields = v.keys()
+
+        check_location_fields(v, fields)
+
+        kwargs = {'{0}__{1}'.format(
+            scope, loc_dict.get(loc_scope)
+            ): v.get(loc_scope)
+            for loc_scope in fields
+            if loc_dict.get(loc_scope) is not None}
+
+        model_name = apps.get_model('awards', model)
+        qs = model_name.objects.filter(**kwargs)
+
+        if or_queryset is not None:
+            or_queryset |= qs
+        else:
+            or_queryset = qs
+
+    return or_queryset
+
+
+def check_location_fields(filters, fields):
```

Comment:
We're passing in `filters` here but it doesn't look like we're using them.

Reply:
Removed filters from function and then when the function is called